src/mxcp/sdk/executor/plugins/python.py

```diff
@@ -93,14 +93,25 @@ class PythonExecutor(ExecutorPlugin):
         >>> executor.shutdown()
     """
     
-    def __init__(self, repo_root: Optional[Path] = None):
+    def __init__(
+        self, 
+        repo_root: Optional[Path] = None,
+        user_config: Optional[Dict] = None,
+        site_config: Optional[Dict] = None,
```

Comment:
This is putting a dependency on site config and user config, which are MXCP concepts (not SDK concepts) into the SDK.